common/lib/dependabot/pull_request_creator/branch_namer/solo_strategy.rb

```diff
@@ -0,0 +1,208 @@
+# frozen_string_literal: true
+
+require "digest"
+
+require "dependabot/metadata_finders"
+
+module Dependabot
+  class PullRequestCreator
+    class BranchNamer
+      class SoloStrategy
```

Comment:
This object represents our current approach to naming. I don't think `Solo` necessarily captures that well, but it stands as a complement to a grouping. I'm open to this changing now or later 🙂